docs/decisions/0005-lti-provider-1.3-spec-support.rst

```diff
@@ -0,0 +1,651 @@
+   graded resources scores. (Ex:
+   https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/lti_provider/signals.py#L40).
+      - Get related graded resource from data received.
+      - Get all assignments related
+      - Increment version value of each assignment
+      - Determine each assignment type and send corresponding task (composite module, or single problem).
+      - Send task to update score for each assignment by sending message back to platform related to resource. (Example: https://github.com/openedx/edx-platform/pull/27411/files#diff-36022deef8607c7a4647c8f2620b4d9ed283d5b41077e966bfd097585e0ebe7cR480)
+
+Approach A: Create new app
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This approach would be similar to the proposed on the `PR
+21435 <https://github.com/openedx/edx-platform/pull/21435>`__). We will
+create a new app (Example: lti1p3_tool), and integrate all logic related
+to LTI 1.3 separated from the existing lti_provider app.
+
+Approach B: Refactor lti_provider app
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We could refactor the existing lti_provider. With this approach we could
+separate the logic of each specification per folder, and keep utilities
+that are used on both specifications on the base folder of the app, here
+is an example of the folder structure:
+
+.. code:: bash
+
+   .
+   ├── __init__.py
+   ├── apps.py
+   ├── admin.py
+   ├── tasks.py
+   ├── urls.py
+   ├── management
+   │   ├── commands
+   │   │   ├── __init__.py
+   │   │   └── ...
+   ├── migrations
+   │   ├── __init__.py
+   │   └── ...
+   ├── models.py
+   │   ├── __init__.py # LtiUser and GradedAssignment model could be store here
+   │   ├── 1p1 # Create LtiUser1p1 using multi table inheritance with LtiUser
+   │   └── 1p3 # Create LtiUser1p3 using multi table inheritance with LtiUser
+   ├── utils
+   │   ├── __init__.py
+   │   ├── users.py
+   │   ├── 1p3 # Store utilities for 1.3 specification
+   │   └── 1p1 # Store utilities for 1.1 specification
+   │       ├── __init__.py
+   │       ├── signature_validator.py
+   │       └── outcomes.py
+   ├── signals.py # Use score_changed_handler for both specifications
+   └── views.py
+       ├── __init__.py
+       ├── 1p1 # Define LTI 1.1 views here
+       └── 1p3 # Define LTI 1.3 views here
+
+LTI 1.3 Open edX community discussion
+-------------------------------------
+
+-  Deep dive into LTI 1.3 in the Open edX platform:
+   https://openedx.org/video/deep-dive-into-lti-1-3-in-the-open-edx-platform/
+-  LTI 1.3 and LTI Advantage:
+   https://discuss.openedx.org/t/lti-1-3-and-lti-advantage/5672
+-  LTI Provider in Nutmeg and future releases?:
+   https://discuss.openedx.org/t/lti-provider-in-nutmeg-and-future-releases/8330/14
+-  A question about LTI provider support in Open edX:
+   https://discuss.openedx.org/t/a-question-about-lti-provider-support-in-open-edx/4866/2
+-  Open edX Slack LTI channel:
+   https://openedx.slack.com/archives/C0GR05YC
+
```

Comment:
Given how this would fit in a new app, is it possible to make that APP live in an openedx-plugin?